pkg/adaptor/cloud/aws/types.go

```diff
@@ -36,6 +38,40 @@ func (i *instanceTypes) Set(value string) error {
 	return nil
 }
 
+// keyValueFlag represents a flag of key-value pairs
+type keyValueFlag map[string]string
```

Comment:
This looks like a generic implementation of parsing `k1=v1,k2=v2`, IMO this should go into some place where we can reuse it everywhere.

Reply:
Yeah, make sense. I'll see where I can put this for generic use